src/components/Login/LoginOTPCard.tsx

```diff
@@ -102,6 +116,17 @@ export const LoginOTPCard: FunctionComponent<LoginOTPCard> = ({
         alert(
           "Encountered an issue obtaining environment information. We've noted this down and are looking into it!"
         );
+      } else if (e instanceof LoginLockedError) {
+        showAlert({
+          ...disabledAccessAlertProps,
+          description: e.message,
+          onOk: () => resetStage()
+        });
+      } else if (e instanceof LoginError) {
+        showAlert({
+          ...invalidEntryAlertProps,
+          description: e.message
+        });
       } else {
         alert(e);
```

Comment:
for this, can throw an error? so that if its unhandled, it will reach the error boundary. 
can also let have sentry capture it too

Reply:
yeah, i think it should be in another pr where we standardise the login error dialogs tho

Reply:
ok nvm i'll do it here!